tests/e2e/validatecluster/validatecluster_test.go

```diff
@@ -0,0 +1,80 @@
+package validatecluster
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/rancher/k3s/tests/e2e"
+)
+
+func Test_E2EClusterValidation(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Create Cluster Test Suite")
+}
+
+const (
+	// Valid nodeOS: generic/ubuntu2004, opensuse/Leap-15.3.x86_6, dweomer/microos.amd64
```

Comment:
I think we should also be testing CentOS/RHEL

Maybe Raspbian? or whatever our K3s on Pi users are using mostly.

Reply:
We currently cannot test CentOS/RHEL due to a lack of RPMS build for each commit by our CI.